python_modules/dagster/setup.py

```diff
@@ -60,8 +60,7 @@ def get_version() -> str:
             "click>=5.0",
             "coloredlogs>=6.1, <=14.0",
             "contextvars; python_version < '3.7'",
-            # https://github.com/dagster-io/dagster/issues/4167
-            "Jinja2<3.0",
+            "Jinja2",
             "markupsafe<=2.0.1",
```

Comment:
would also want to unpin markupsafe... but more likely, it will be necessary to set a bottom pin on both to get a working install...